app/models/history.rb

```diff
@@ -463,30 +466,42 @@ def self.clear_follow_up_flag(history)
   end
 
   private_class_method def self.compose_message(history, field)
-    creator = history[:household_status] == :patient ? 'User' : 'System'
+    creator = history[:initiator_id].nil? ? 'System' : 'User'
```

Comment:
I've noticed that in some messages, we now give a message that looks like this:
![image](https://user-images.githubusercontent.com/52747882/124812790-13e3ff00-df32-11eb-848c-bfc79e9f3a02.png)
 where we indicate that the `System` made the change, but then we also list the user that made the change. I'm actually not quite sure what is correct, saying that a `User` or `System` made the change, but I think the two pieces of information should be consistent.